apps/communications/contacts/test/unit/views/multiple_select_test.js

```diff
@@ -0,0 +1,66 @@
+/* global Contacts, MockContactsList, MockLazyLoader, MockMultipleSelectHTML,
+          MockMatcher */
+'use strict';
+
+requireApp('communications/contacts/js/views/multiple_select.js');
+requireApp('communications/contacts/test/unit/mock_contacts_list.js');
+require('/shared/test/unit/mocks/mock_lazy_loader.js');
+requireApp('communications/contacts/test/unit/mock_multiple_select.js.html');
+requireApp('communications/contacts/test/unit/mock_contacts_match.js');
+
+window.contacts = window.contacts || {};
+
+suite('MultipleSelect view', function() {
+  var realLazyLoader,
+      realMatcher;
+
+  suiteSetup(function() {
+    realLazyLoader = window.LazyLoader;
+    window.LazyLoader = MockLazyLoader;
+
+    realMatcher = window.contacts.Matcher;
+    window.contacts.Matcher = MockMatcher;
+  });
+
+  suiteTeardown(function() {
+    window.LazyLoader = realLazyLoader;
+    window.contacts = realMatcher;
+  });
+
+  setup(function() {
+    document.body.innerHTML = MockMultipleSelectHTML;
+  });
+
+  test('Adding contacts to the view', function() {
+    Contacts.MultipleSelect.render(MockContactsList(), {});
+    assert.equal(document.querySelectorAll('li').length, 3);
+  });
+
+  test('Loading contacts from cursor', function() {
+    var howManyContacts = 4,
+        alreadyLoaded = 0,
+        onsuccessCallback;
+
+    Contacts.MultipleSelect.render([], {
+      set onsuccess(callback) {
```

Comment:
please separate out the dummy cursor from the code of the test case avoiding any coupling between both